src/runner.rs

```diff
@@ -224,4 +228,38 @@ When your transaction is complete, enter 'commit' or 'abort' as appropriate."#,
         }
         Ok(())
     }
+
+    pub(crate) async fn handle_status(&self) -> Result<()> {
+        // TODO: Return latency information from recent commands if we're able to capture it.
+        self.deps.ui.println(&format!("Current region: {}", self.deps.env.region().value.name()));
+        self.deps.ui.println(&format!("Current ledger: {}", self.deps.driver.ledger_name()));
+        self.deps.ui.println(&format!("Shell version: {}", env!("CARGO_PKG_VERSION")));
+        Ok(())
+    }
+
+    pub(crate) async fn handle_ping(&self) -> Result<()> {
+        let region = self.deps.env.region().value.name().to_string();
+        let mut request_url = Url::parse(&format!("https://session.qldb.{}.amazonaws.com/ping", region))?;
+
+        let qldb_session_endpoint = self.deps.env.qldb_session_endpoint().value;
+        if qldb_session_endpoint.is_some() {
+            request_url = qldb_session_endpoint.unwrap().join("ping")?;
+        }
```

Comment:
I think you should try rewrite this using a match statement such that you don't need the unwrap.

Reply:
Updated to use `match` statement